resources/views/components/tables/desktop/wallets.blade.php

```diff
@@ -1,30 +1,68 @@
-<x-ark-tables.table sticky class="hidden w-full md:block">
-    <thead>
-        <tr>
+<x-tables.encapsulated-table sticky class="hidden w-full md:block">
```

Comment:
Can we do this or is it some magic component? :thinking: 

```suggestion
@props([
    'wallets',
    'hideVoting' => false,
    'useVoteWeight' => false, // change the `isset($userVoteWeight)` checks
])

<x-tables.encapsulated-table sticky class="hidden w-full md:block">
```

Reply:
props will work, just kept it in line with what we had before but will adjust it to make it prettier and easier to work with